catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/RTL_BricksTest.java

```diff
@@ -0,0 +1,163 @@
+	private static final String DEFAULT_TEST_PROJECT_NAME = "testProject";
+	private static final Locale ARABICLOCALE = new Locale("ar");
+
+	@Rule
+	public ActivityTestRule<MainMenuActivity> mActivityRule = new ActivityTestRule<>(MainMenuActivity.class);
+
+	@Before
+	public void setUp() throws Exception {
+		createTestProject(DEFAULT_TEST_PROJECT_NAME);
+	}
+
+	@Test
+	public void assertBricksLayoutAndTextDirectionAreRTL() {
+		assertThat("LayoutDirection is not RTL", true, is(isRTL()));
+		assertThat("current Locale is not Arabic", Locale.getDefault().getDisplayLanguage(), is(ARABICLOCALE.getDisplayLanguage()));
+		onView(withText(R.string.main_menu_programs)).perform(click());
+		onData(anything()).inAdapterView(withId(android.R.id.list)).atPosition(0).perform(click());
+		onView(withId(R.id.spritelist_item_background)).perform(click());
+		onView(withId(R.id.program_menu_button_scripts)).perform(click());
+
+		onView(withId(R.id.brick_when_started_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_when_started)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_hide_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_hide)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_show_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_show)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_set_size_to_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_set_size_to)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_go_back_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_go_back)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_go_to_front_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_come_to_front)).check(TextDirectionAssertions.isTextDirectionRTL());
+
+		onView(withId(R.id.brick_place_at_layout)).check(LayoutDirectionAssertions.isLayoutDirectionRTL());
+		onView(withText(R.string.brick_place_at)).check(TextDirectionAssertions.isTextDirectionRTL());
+	}
+
+	public static List<Brick> createTestProject(String projectName) {
+		int xPosition = 457;
+		int yPosition = 598;
+		double size = 0.8;
+
+		Project project = new Project(null, projectName);
+		Sprite firstSprite = new SingleSprite("cat");
+
+		Script testScript = new StartScript();
+
+		ArrayList<Brick> brickList = new ArrayList<>();
+		brickList.add(new HideBrick());
+		brickList.add(new ShowBrick());
+		brickList.add(new SetSizeToBrick(size));
+		brickList.add(new GoNStepsBackBrick(1));
+		brickList.add(new ComeToFrontBrick());
+		brickList.add(new PlaceAtBrick(xPosition, yPosition));
+
+		for (Brick brick : brickList) {
```

Comment:
but why do this instead of adding the bricks directly to the startScript?
and also, please rename testScript to startScript